// ramda/src/index.js
module.exports = {};
module.exports.F = /*#__PURE__*/require('ramda/src/F');
module.exports.T = /*#__PURE__*/require('ramda/src/T');
module.exports.__ = /*#__PURE__*/require('ramda/src/__');
module.exports.add = /*#__PURE__*/require('ramda/src/add');
module.exports.addIndex = /*#__PURE__*/require('ramda/src/addIndex');
module.exports.adjust = /*#__PURE__*/require('ramda/src/adjust');
module.exports.all = /*#__PURE__*/require('ramda/src/all');
module.exports.allPass = /*#__PURE__*/require('ramda/src/allPass');
module.exports.always = /*#__PURE__*/require('ramda/src/always');
module.exports.and = /*#__PURE__*/require('ramda/src/and');
module.exports.any = /*#__PURE__*/require('ramda/src/any');
module.exports.anyPass = /*#__PURE__*/require('ramda/src/anyPass');
module.exports.ap = /*#__PURE__*/require('ramda/src/ap');
module.exports.aperture = /*#__PURE__*/require('ramda/src/aperture');
module.exports.append = /*#__PURE__*/require('ramda/src/append');
module.exports.apply = /*#__PURE__*/require('ramda/src/apply');
module.exports.applySpec = /*#__PURE__*/require('ramda/src/applySpec');
module.exports.applyTo = /*#__PURE__*/require('ramda/src/applyTo');
module.exports.ascend = /*#__PURE__*/require('ramda/src/ascend');
module.exports.assoc = /*#__PURE__*/require('ramda/src/assoc');
module.exports.assocPath = /*#__PURE__*/require('ramda/src/assocPath');
module.exports.binary = /*#__PURE__*/require('ramda/src/binary');
module.exports.bind = /*#__PURE__*/require('ramda/src/bind');
module.exports.both = /*#__PURE__*/require('ramda/src/both');
module.exports.call = /*#__PURE__*/require('ramda/src/call');
module.exports.chain = /*#__PURE__*/require('ramda/src/chain');
module.exports.clamp = /*#__PURE__*/require('ramda/src/clamp');
module.exports.clone = /*#__PURE__*/require('ramda/src/clone');
module.exports.comparator = /*#__PURE__*/require('ramda/src/comparator');
module.exports.complement = /*#__PURE__*/require('ramda/src/complement');
module.exports.compose = /*#__PURE__*/require('ramda/src/compose');
module.exports.composeK = /*#__PURE__*/require('ramda/src/composeK');
module.exports.composeP = /*#__PURE__*/require('ramda/src/composeP');
module.exports.concat = /*#__PURE__*/require('ramda/src/concat');
module.exports.cond = /*#__PURE__*/require('ramda/src/cond');
module.exports.construct = /*#__PURE__*/require('ramda/src/construct');
module.exports.constructN = /*#__PURE__*/require('ramda/src/constructN');
module.exports.contains = /*#__PURE__*/require('ramda/src/contains');
module.exports.converge = /*#__PURE__*/require('ramda/src/converge');
module.exports.countBy = /*#__PURE__*/require('ramda/src/countBy');
module.exports.curry = /*#__PURE__*/require('ramda/src/curry');
module.exports.curryN = /*#__PURE__*/require('ramda/src/curryN');
module.exports.dec = /*#__PURE__*/require('ramda/src/dec');
module.exports.defaultTo = /*#__PURE__*/require('ramda/src/defaultTo');
module.exports.descend = /*#__PURE__*/require('ramda/src/descend');
module.exports.difference = /*#__PURE__*/require('ramda/src/difference');
module.exports.differenceWith = /*#__PURE__*/require('ramda/src/differenceWith');
module.exports.dissoc = /*#__PURE__*/require('ramda/src/dissoc');
module.exports.dissocPath = /*#__PURE__*/require('ramda/src/dissocPath');
module.exports.divide = /*#__PURE__*/require('ramda/src/divide');
module.exports.drop = /*#__PURE__*/require('ramda/src/drop');
module.exports.dropLast = /*#__PURE__*/require('ramda/src/dropLast');
module.exports.dropLastWhile = /*#__PURE__*/require('ramda/src/dropLastWhile');
module.exports.dropRepeats = /*#__PURE__*/require('ramda/src/dropRepeats');
module.exports.dropRepeatsWith = /*#__PURE__*/require('ramda/src/dropRepeatsWith');
module.exports.dropWhile = /*#__PURE__*/require('ramda/src/dropWhile');
module.exports.either = /*#__PURE__*/require('ramda/src/either');
module.exports.empty = /*#__PURE__*/require('ramda/src/empty');
module.exports.endsWith = /*#__PURE__*/require('ramda/src/endsWith');
module.exports.eqBy = /*#__PURE__*/require('ramda/src/eqBy');
module.exports.eqProps = /*#__PURE__*/require('ramda/src/eqProps');
module.exports.equals = /*#__PURE__*/require('ramda/src/equals');
module.exports.evolve = /*#__PURE__*/require('ramda/src/evolve');
module.exports.filter = /*#__PURE__*/require('ramda/src/filter');
module.exports.find = /*#__PURE__*/require('ramda/src/find');
module.exports.findIndex = /*#__PURE__*/require('ramda/src/findIndex');
module.exports.findLast = /*#__PURE__*/require('ramda/src/findLast');
module.exports.findLastIndex = /*#__PURE__*/require('ramda/src/findLastIndex');
module.exports.flatten = /*#__PURE__*/require('ramda/src/flatten');
module.exports.flip = /*#__PURE__*/require('ramda/src/flip');
module.exports.forEach = /*#__PURE__*/require('ramda/src/forEach');
module.exports.forEachObjIndexed = /*#__PURE__*/require('ramda/src/forEachObjIndexed');
module.exports.fromPairs = /*#__PURE__*/require('ramda/src/fromPairs');
module.exports.groupBy = /*#__PURE__*/require('ramda/src/groupBy');
module.exports.groupWith = /*#__PURE__*/require('ramda/src/groupWith');
module.exports.gt = /*#__PURE__*/require('ramda/src/gt');
module.exports.gte = /*#__PURE__*/require('ramda/src/gte');
module.exports.has = /*#__PURE__*/require('ramda/src/has');
module.exports.hasIn = /*#__PURE__*/require('ramda/src/hasIn');
module.exports.head = /*#__PURE__*/require('ramda/src/head');
module.exports.identical = /*#__PURE__*/require('ramda/src/identical');
module.exports.identity = /*#__PURE__*/require('ramda/src/identity');
module.exports.ifElse = /*#__PURE__*/require('ramda/src/ifElse');
module.exports.inc = /*#__PURE__*/require('ramda/src/inc');
module.exports.indexBy = /*#__PURE__*/require('ramda/src/indexBy');
module.exports.indexOf = /*#__PURE__*/require('ramda/src/indexOf');
module.exports.init = /*#__PURE__*/require('ramda/src/init');
module.exports.innerJoin = /*#__PURE__*/require('ramda/src/innerJoin');
module.exports.insert = /*#__PURE__*/require('ramda/src/insert');
module.exports.insertAll = /*#__PURE__*/require('ramda/src/insertAll');
module.exports.intersection = /*#__PURE__*/require('ramda/src/intersection');
module.exports.intersperse = /*#__PURE__*/require('ramda/src/intersperse');
module.exports.into = /*#__PURE__*/require('ramda/src/into');
module.exports.invert = /*#__PURE__*/require('ramda/src/invert');
module.exports.invertObj = /*#__PURE__*/require('ramda/src/invertObj');
module.exports.invoker = /*#__PURE__*/require('ramda/src/invoker');
module.exports.is = /*#__PURE__*/require('ramda/src/is');
module.exports.isEmpty = /*#__PURE__*/require('ramda/src/isEmpty');
module.exports.isNil = /*#__PURE__*/require('ramda/src/isNil');
module.exports.join = /*#__PURE__*/require('ramda/src/join');
module.exports.juxt = /*#__PURE__*/require('ramda/src/juxt');
module.exports.keys = /*#__PURE__*/require('ramda/src/keys');
module.exports.keysIn = /*#__PURE__*/require('ramda/src/keysIn');
module.exports.last = /*#__PURE__*/require('ramda/src/last');
module.exports.lastIndexOf = /*#__PURE__*/require('ramda/src/lastIndexOf');
module.exports.length = /*#__PURE__*/require('ramda/src/length');
module.exports.lens = /*#__PURE__*/require('ramda/src/lens');
module.exports.lensIndex = /*#__PURE__*/require('ramda/src/lensIndex');
module.exports.lensPath = /*#__PURE__*/require('ramda/src/lensPath');
module.exports.lensProp = /*#__PURE__*/require('ramda/src/lensProp');
module.exports.lift = /*#__PURE__*/require('ramda/src/lift');
module.exports.liftN = /*#__PURE__*/require('ramda/src/liftN');
module.exports.lt = /*#__PURE__*/require('ramda/src/lt');
module.exports.lte = /*#__PURE__*/require('ramda/src/lte');
module.exports.map = /*#__PURE__*/require('ramda/src/map');
module.exports.mapAccum = /*#__PURE__*/require('ramda/src/mapAccum');
module.exports.mapAccumRight = /*#__PURE__*/require('ramda/src/mapAccumRight');
module.exports.mapObjIndexed = /*#__PURE__*/require('ramda/src/mapObjIndexed');
module.exports.match = /*#__PURE__*/require('ramda/src/match');
module.exports.mathMod = /*#__PURE__*/require('ramda/src/mathMod');
module.exports.max = /*#__PURE__*/require('ramda/src/max');
module.exports.maxBy = /*#__PURE__*/require('ramda/src/maxBy');
module.exports.mean = /*#__PURE__*/require('ramda/src/mean');
module.exports.median = /*#__PURE__*/require('ramda/src/median');
module.exports.memoize = /*#__PURE__*/require('ramda/src/memoize');
module.exports.memoizeWith = /*#__PURE__*/require('ramda/src/memoizeWith');
module.exports.merge = /*#__PURE__*/require('ramda/src/merge');
module.exports.mergeAll = /*#__PURE__*/require('ramda/src/mergeAll');
module.exports.mergeDeepLeft = /*#__PURE__*/require('ramda/src/mergeDeepLeft');
module.exports.mergeDeepRight = /*#__PURE__*/require('ramda/src/mergeDeepRight');
module.exports.mergeDeepWith = /*#__PURE__*/require('ramda/src/mergeDeepWith');
module.exports.mergeDeepWithKey = /*#__PURE__*/require('ramda/src/mergeDeepWithKey');
module.exports.mergeWith = /*#__PURE__*/require('ramda/src/mergeWith');
module.exports.mergeWithKey = /*#__PURE__*/require('ramda/src/mergeWithKey');
module.exports.min = /*#__PURE__*/require('ramda/src/min');
module.exports.minBy = /*#__PURE__*/require('ramda/src/minBy');
module.exports.modulo = /*#__PURE__*/require('ramda/src/modulo');
module.exports.multiply = /*#__PURE__*/require('ramda/src/multiply');
module.exports.nAry = /*#__PURE__*/require('ramda/src/nAry');
module.exports.negate = /*#__PURE__*/require('ramda/src/negate');
module.exports.none = /*#__PURE__*/require('ramda/src/none');
module.exports.not = /*#__PURE__*/require('ramda/src/not');
module.exports.nth = /*#__PURE__*/require('ramda/src/nth');
module.exports.nthArg = /*#__PURE__*/require('ramda/src/nthArg');
module.exports.o = /*#__PURE__*/require('ramda/src/o');
module.exports.objOf = /*#__PURE__*/require('ramda/src/objOf');
module.exports.of = /*#__PURE__*/require('ramda/src/of');
module.exports.omit = /*#__PURE__*/require('ramda/src/omit');
module.exports.once = /*#__PURE__*/require('ramda/src/once');
module.exports.or = /*#__PURE__*/require('ramda/src/or');
module.exports.over = /*#__PURE__*/require('ramda/src/over');
module.exports.pair = /*#__PURE__*/require('ramda/src/pair');
module.exports.partial = /*#__PURE__*/require('ramda/src/partial');
module.exports.partialRight = /*#__PURE__*/require('ramda/src/partialRight');
module.exports.partition = /*#__PURE__*/require('ramda/src/partition');
module.exports.path = /*#__PURE__*/require('ramda/src/path');
module.exports.pathEq = /*#__PURE__*/require('ramda/src/pathEq');
module.exports.pathOr = /*#__PURE__*/require('ramda/src/pathOr');
module.exports.pathSatisfies = /*#__PURE__*/require('ramda/src/pathSatisfies');
module.exports.pick = /*#__PURE__*/require('ramda/src/pick');
module.exports.pickAll = /*#__PURE__*/require('ramda/src/pickAll');
module.exports.pickBy = /*#__PURE__*/require('ramda/src/pickBy');
module.exports.pipe = /*#__PURE__*/require('ramda/src/pipe');
module.exports.pipeK = /*#__PURE__*/require('ramda/src/pipeK');
module.exports.pipeP = /*#__PURE__*/require('ramda/src/pipeP');
module.exports.pluck = /*#__PURE__*/require('ramda/src/pluck');
module.exports.prepend = /*#__PURE__*/require('ramda/src/prepend');
module.exports.product = /*#__PURE__*/require('ramda/src/product');
module.exports.project = /*#__PURE__*/require('ramda/src/project');
module.exports.prop = /*#__PURE__*/require('ramda/src/prop');
module.exports.propEq = /*#__PURE__*/require('ramda/src/propEq');
module.exports.propIs = /*#__PURE__*/require('ramda/src/propIs');
module.exports.propOr = /*#__PURE__*/require('ramda/src/propOr');
module.exports.propSatisfies = /*#__PURE__*/require('ramda/src/propSatisfies');
module.exports.props = /*#__PURE__*/require('ramda/src/props');
module.exports.range = /*#__PURE__*/require('ramda/src/range');
module.exports.reduce = /*#__PURE__*/require('ramda/src/reduce');
module.exports.reduceBy = /*#__PURE__*/require('ramda/src/reduceBy');
module.exports.reduceRight = /*#__PURE__*/require('ramda/src/reduceRight');
module.exports.reduceWhile = /*#__PURE__*/require('ramda/src/reduceWhile');
module.exports.reduced = /*#__PURE__*/require('ramda/src/reduced');
module.exports.reject = /*#__PURE__*/require('ramda/src/reject');
module.exports.remove = /*#__PURE__*/require('ramda/src/remove');
module.exports.repeat = /*#__PURE__*/require('ramda/src/repeat');
module.exports.replace = /*#__PURE__*/require('ramda/src/replace');
module.exports.reverse = /*#__PURE__*/require('ramda/src/reverse');
module.exports.scan = /*#__PURE__*/require('ramda/src/scan');
module.exports.sequence = /*#__PURE__*/require('ramda/src/sequence');
module.exports.set = /*#__PURE__*/require('ramda/src/set');
module.exports.slice = /*#__PURE__*/require('ramda/src/slice');
module.exports.sort = /*#__PURE__*/require('ramda/src/sort');
module.exports.sortBy = /*#__PURE__*/require('ramda/src/sortBy');
module.exports.sortWith = /*#__PURE__*/require('ramda/src/sortWith');
module.exports.split = /*#__PURE__*/require('ramda/src/split');
module.exports.splitAt = /*#__PURE__*/require('ramda/src/splitAt');
module.exports.splitEvery = /*#__PURE__*/require('ramda/src/splitEvery');
module.exports.splitWhen = /*#__PURE__*/require('ramda/src/splitWhen');
module.exports.startsWith = /*#__PURE__*/require('ramda/src/startsWith');
module.exports.subtract = /*#__PURE__*/require('ramda/src/subtract');
module.exports.sum = /*#__PURE__*/require('ramda/src/sum');
module.exports.symmetricDifference = /*#__PURE__*/require('ramda/src/symmetricDifference');
module.exports.symmetricDifferenceWith = /*#__PURE__*/require('ramda/src/symmetricDifferenceWith');
module.exports.tail = /*#__PURE__*/require('ramda/src/tail');
module.exports.take = /*#__PURE__*/require('ramda/src/take');
module.exports.takeLast = /*#__PURE__*/require('ramda/src/takeLast');
module.exports.takeLastWhile = /*#__PURE__*/require('ramda/src/takeLastWhile');
module.exports.takeWhile = /*#__PURE__*/require('ramda/src/takeWhile');
module.exports.tap = /*#__PURE__*/require('ramda/src/tap');
module.exports.test = /*#__PURE__*/require('ramda/src/test');
module.exports.times = /*#__PURE__*/require('ramda/src/times');
module.exports.toLower = /*#__PURE__*/require('ramda/src/toLower');
module.exports.toPairs = /*#__PURE__*/require('ramda/src/toPairs');
module.exports.toPairsIn = /*#__PURE__*/require('ramda/src/toPairsIn');
module.exports.toString = /*#__PURE__*/require('ramda/src/toString');
module.exports.toUpper = /*#__PURE__*/require('ramda/src/toUpper');
module.exports.transduce = /*#__PURE__*/require('ramda/src/transduce');
module.exports.transpose = /*#__PURE__*/require('ramda/src/transpose');
module.exports.traverse = /*#__PURE__*/require('ramda/src/traverse');
module.exports.trim = /*#__PURE__*/require('ramda/src/trim');
module.exports.tryCatch = /*#__PURE__*/require('ramda/src/tryCatch');
module.exports.type = /*#__PURE__*/require('ramda/src/type');
module.exports.unapply = /*#__PURE__*/require('ramda/src/unapply');
module.exports.unary = /*#__PURE__*/require('ramda/src/unary');
module.exports.uncurryN = /*#__PURE__*/require('ramda/src/uncurryN');
module.exports.unfold = /*#__PURE__*/require('ramda/src/unfold');
module.exports.union = /*#__PURE__*/require('ramda/src/union');
module.exports.unionWith = /*#__PURE__*/require('ramda/src/unionWith');
module.exports.uniq = /*#__PURE__*/require('ramda/src/uniq');
module.exports.uniqBy = /*#__PURE__*/require('ramda/src/uniqBy');
module.exports.uniqWith = /*#__PURE__*/require('ramda/src/uniqWith');
module.exports.unless = /*#__PURE__*/require('ramda/src/unless');
module.exports.unnest = /*#__PURE__*/require('ramda/src/unnest');
module.exports.until = /*#__PURE__*/require('ramda/src/until');
module.exports.update = /*#__PURE__*/require('ramda/src/update');
module.exports.useWith = /*#__PURE__*/require('ramda/src/useWith');
module.exports.values = /*#__PURE__*/require('ramda/src/values');
module.exports.valuesIn = /*#__PURE__*/require('ramda/src/valuesIn');
module.exports.view = /*#__PURE__*/require('ramda/src/view');
module.exports.when = /*#__PURE__*/require('ramda/src/when');
module.exports.where = /*#__PURE__*/require('ramda/src/where');
module.exports.whereEq = /*#__PURE__*/require('ramda/src/whereEq');
module.exports.without = /*#__PURE__*/require('ramda/src/without');
module.exports.xprod = /*#__PURE__*/require('ramda/src/xprod');
module.exports.zip = /*#__PURE__*/require('ramda/src/zip');
module.exports.zipObj = /*#__PURE__*/require('ramda/src/zipObj');
module.exports.zipWith = /*#__PURE__*/require('ramda/src/zipWith');
